### ISSUE TITLE
[Bug]: Modal Gets Scrolled Down If Changing the Width of a Widget Inside It

### ISSUE THREAD

**Issue by ame-appsmith** [appsmithorg/appsmith#23105]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When a user has a scrollable Modal widget and modifies the width of another widget nested inside the modal, it gets scrolled down to the bottom. 

### Steps To Reproduce

1. Add a Modal to the canvas.
2. Add several widgets inside of it so that it becomes scrollable.
3. Modify the width of a widget from the top.

### Environment

Production

### Issue video log

https://www.loom.com/share/2d923c8026bb4432b8116467898bf9de

### Version

Cloud/Self Hosted

**Comment by shanu798** [comment]
yes, I am facing the same issue, whenever i tried to adjust height and width of any widgets, it gets scrolled down the modal

**Comment by rahulbarwal** [comment]
Reopening as the issue is still persistent on app.appsmith.com

**Comment by shadabbuchh** [comment]
> Reopening as the issue is still persistent on app.appsmith.com

@rahulbarwal do we plan on fixing this issue in the upcoming sprints? Asking since a similar behaviour this was reported by a user recently.

**Comment by rahulbarwal** [comment]
@shadabbuchh this is not a priority for now. Do you see a usecase where this is blocking?

**Comment by shadabbuchh** [comment]
> @shadabbuchh this is not a priority for now. Do you see a usecase where this is blocking?

@rahulbarwal This is not a blocker, but it was marked as a high priority by a customer in our Oracle app. Hence, I wanted to confirm.
